### DISCUSSION TITLE
How do I change the theme?

### DISCUSSION THREAD

**Discussion by rocheston** [DustinBrett/daedalOS#305] [Feature Requests]
Is there a way to change the default Windows theme?

**Comment by DustinBrett** [comment]
Currently there is not as I haven't made any other theme. It's on my to do post-it wall to make the light theme, which will also entail an overhaul of the theme code to be more centralized. Until then the only way would be to fork the code and make manual adjustments. Thanks for asking!

**Comment by rocheston** [comment]
I tried to change file manager window background color

  window: {
    background: "#2F343F",
    outline: "hsla(0, 0%, 25%, 75%)",
    outlineInactive: "hsla(0, 0%, 30%, 100%)",
    shadow: "0 0 12px 0 rgba(0, 0, 0, 50%)",
    shadowInactive: "0 0 8px 0 rgba(0, 0, 0, 50%)",
  },
};

This does not work. I can change the other colors though like titlebar colors.

**Reply by DustinBrett** [reply]
Ideally all colors will be centralized in `styles/defaultTheme/colors.ts`. Every color you would change should be there or be moved there.
